bazel/repositories.bzl

```diff
@@ -81,3 +82,11 @@ def com_github_gabime_spdlog():
     build_file = "//bazel:spdlog.BUILD",
     urls = ["https://github.com/gabime/spdlog/archive/v1.7.0.tar.gz"]
   )
+
+def com_google_abseil():
+  http_archive(
+    name = "com_github_abseil-cpp",
```

Comment:
What does this repo for?

Reply:
This is common operation library used in Chromium (also in Envoy). I used this to introduce endsWith function. (Unfortunately, C++ STL doesn't have functionality that other languages have. So we should use these common libraries)

Reply:
Got it.